src/main/scala/is/hail/methods/Aggregators.scala

```diff
@@ -570,3 +571,27 @@ class KeyedAggregator[T, K](aggregator: TypedAggregator[T]) extends TypedAggrega
 
   def copy() = new KeyedAggregator(aggregator.copy())
 }
+
+class VirtualDSAggregator(xmin: Double, xmax: Double, ymin: Double, ymax: Double, nDivisions: Int) extends TypedAggregator[IndexedSeq[Row]] {
```

Comment:
You should check and call fatal here if the constructor arguments (xmin, etc.) are invalid (e.g. xmax < xmin, nDivisions < 0, etc.)  We do the same thing in RegionValueHistogramAggregator, for example.